### DISCUSSION TITLE
locked block rewards are not vesting anymore

### DISCUSSION THREAD

**Discussion by leobelive** [filecoin-project/lotus#6032] [Storage Provider] [answered]
**Describe the bug**

in my miner account ,  the  'locked Rewards' has stop release about a week, i do not know why?  
from command 'lotus-miner info' ,  the part of  'Vesting' do not change ,  it is happen to a week ago , i drop 20T power for  some reason ，but it is repaired by next day 。after this ， the 'Vesting' has not changed !  and  mined info never appeared .

**Version (run `lotus version`):**
first  -> lotus 1.5.0  
then upgrad to  ->  1.6.0 

**To Reproduce**
in these days,  i restart  miner job , but it is  no use , the issue still exist 

**Expected behavior**
how to do ,  let the  part of 'Vesting'  begin to release ! 

**Logs**
miner logs is normal , just  same info  like below 
2021-04-14T01:18:06.064+0800	INFO	miner	miner/miner.go:383	Time delta between now and our mining base: 6s (nulls: 0)

 
**Screenshots**
![image](https://user-images.githubusercontent.com/4366253/114592068-20531580-9cbd-11eb-88b5-924cfc0207a4.png)

**Additional context**
miner id  is  f0126698

**Comment by de-authority** [comment]
Does your problem  solved. I can see you locked fil is increase.

**Comment by de-authority** [comment, marked as answer]
This is caused by missing window post . and get penalty. it will be taken off the locked rewards aka vesting account and alter the original payout schedule.
answer  by @Patrick.
and you can always use ` lotus state read-state [command options] [actorAddress]`  to  get vesting schedule

**Comment by LexLuthr** [comment]
@leobelive Do you still need help with this problem or were you able to fix it with the help of above answers?
